src/akamai/js/akamaiHD_flash.js

```diff
@@ -201,8 +211,29 @@ require("../../../html5-common/js/utils/constants.js");
      * @returns {boolean} True or false indicating success
      */
     this.setVideoUrl = function(url, encoding) {
-      /* Should be set to false if stream URL changes */
-      loaded = false;
+      self=this;
```

Comment:
Why are you setting self in setVideo Url?  Why not just set it when self is defined?